sdk/program/src/poseidon_bn254.rs

```diff
@@ -0,0 +1,224 @@
+use thiserror::Error;
```

Comment:
Can we add a doc here regarding the Poseidon parameters? Copying the first paragraph of the "Summary of Changes" first paragraph in the PR description would be sufficient.